lib/pages/course_card.dart

```diff
@@ -0,0 +1,180 @@
+import 'package:flutter/material.dart';
+
+List<CardItem> items = [
+  const CardItem(
+      image: "card1.png",
+      title: 'Art & Humanities',
+      subtitle: 'Draw & Paint Arts',
+      duration: '2Hr 20Mins'),
```

Comment:
duration instead of using a use Duration class and later convert into this text

Reply:
done